### PR TITLE
feat(bower): add /dt support

### PR DESCRIPTION
**Pull request by mgcrea**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -3783,6 +3783,45 @@ cache(function(data, match, sendBadge, request) {
...
+      if (total === 0) {
+        badgeData.colorscheme = 'red';
+      } else if (total < 100) {
+        badgeData.colorscheme = 'yellow';
+      } else if (total < 1000) {
+        badgeData.colorscheme = 'yellowgreen';
+      } else if (total < 10000) {
+        badgeData.colorscheme = 'green';
+      } else {
+        badgeData.colorscheme = 'brightgreen';
```
Comment (espadrine) [inline server.js]: I believe you could shorten this piece to a single line by using `floorCountColor()`.

**Comment by espadrine** [comment]
Thank you for your contribution! Sorry it took me so long to get to it. It should be much faster after this one change.

**Comment by paulmelnikow** [comment]
Hi, I'd like to get this merged. Would you be willing to [add tests](https://github.com/badges/shields/tree/master/service-tests) and address the review comment?

`index.html` has been removed in master, so you can remove it when you merge master into your branch.

**Comment by paulmelnikow** [comment]
If anyone would like to adopt this PR, please feel free to do so: grab the branch, add more commits addressing the comments, and open a new PR referencing this one.

**Pull request closed without merging**